src/styles/_common.scss

```
@@ -10,12 +10,12 @@ body {
 
 .row {
   margin: 50px auto;
-  display: flex;
+  display: grid;
   justify-content: center;
-  
-  @media (max-width: 918px) {
-    margin-top: 0;
-    flex-wrap: wrap;
-    align-items: space-evenly;
+  grid-template-columns: repeat(auto-fit, 200px);
+  grid-gap: 48px;
```

In Figma mockup, you can see than row-gap is equal 50px
```suggestion
  grid-gap: 50 48px;
```